tsconfig.json

```diff
@@ -7,6 +7,11 @@
 		"declaration": true,
 		"pretty": true,
 		"newLine": "lf",
+		"lib": [
```

Comment:
Had to set this to be able to use `SharedArrayBuffer`. Should we use `es2015` instead because that's our main target?

Reply:
> Should we use es2015 instead because that's our main target?

Yes